### PR TITLE
add pprint library to compliance tests

### PR DESCRIPTION
**Pull request by yisraelU**
If we are ok adding another library, Pprint is helpfult to present as clear evidence as possible of inequality between values when using the Compliance tests

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -735,7 +735,8 @@ lazy val complianceTests = projectMatrix
       ce3 ++ Seq(
         Dependencies.Http4s.circe.value,
         Dependencies.Http4s.client.value,
-        Dependencies.Weaver.cats.value % Test
+        Dependencies.Weaver.cats.value % Test,
+        "com.lihaoyi" %% "pprint" % "0.8.1"
```
Comment (Baccata) [inline build.sbt:739]: make a line for this in the `Dependencies` object. Also http://sca.la/percent

Comment (daddykotex) [reply build.sbt:739]: lolll he's got a shorten url for it hahahaha

File: build.sbt [hunk 2]
```diff
@@ -32,7 +32,7 @@ Global / licenses := Seq(
 sonatypeCredentialHost := "s01.oss.sonatype.org"
 
 ThisBuild / version := {
-  if (!sys.env.contains("CI")) "dev-SNAPSHOT"
+  if (!sys.env.contains("CI")) "dev-SNAPSHOT1"
```
[the remark this answers is not in the document]
Comment (yisraelU) [reply build.sbt:35]: whoops

[the remark this answers is not in the document]
Comment (Baccata) [reply build.sbt:35]: Still not resolved.

File: modules/compliance-tests/src/smithy4s/compliancetests/internals/Assertions.scala [hunk 3]
```diff
@@ -45,7 +45,21 @@ private[internals] object assert {
     if (expected == actual) {
       success
     } else {
-      fail(s"Actual value: $actual was not equal to $expected.")
+      fail(
+        s"Actual value: ${pprint.apply(actual)} was not equal to ${pprint.apply(expected)}."
```
Comment (Baccata) [inline modules/compliance-tests/src/smithy4s/compliancetests/internals/Assertions.scala:49]: 👍

**Comment by daddykotex** [comment]
I received an email a commit removing the `1` suffix but I can't find it lol

File: build.sbt (hunk 2 above)
Comment (Baccata) [inline build.sbt:35]:
```suggestion
  if (!sys.env.contains("CI")) "dev-SNAPSHOT"
```

**Review by Baccata — APPROVED** [review]

**Pull request merged**